news/4.34/platform.html

```diff
@@ -181,6 +181,29 @@ <h2>General Updates </h2>
         </p>
       </td>
     </tr>
+    
+    <tr id="edge-browser-improvements"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466 -->
+      <td class="title">Several Improvements made to the Edge Browser (Windows only)</td>
+      <td class="content">
+        <p>
+          In an effort to make it the default browser for Windows, several issues that affected the Edge browser 
+          have been addressed in this release.
+        </p>
+        <p>
+          Edge still <b>isn't</b> the default browser but it is mature enough that it can be activated and used already.
+          In order to replace IE with Edge as the browser of choice in Windows, you have to set the following parameter 
+          in your <strong>eclipse.ini</strong>:
+        </p>
+        <p>
+          <b>-Dorg.eclipse.swt.browser.DefaultType=edge</b>
+        </p>
```

Comment:
I would propose to take the chance and refer to the "monitor-specific scaling" news, as enabkung that feature will also enable Edge without any need to modify ini files.

Reply:
Done